runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -73,51 +95,84 @@ default boolean enableCustomChrome()
 		keyName = "gameAlwaysOnTop",
 		name = "Enable client always on top",
 		description = "The game will always be on the top of the screen",
-		position = 4
+		position = 15
 	)
 	default boolean gameAlwaysOnTop()
 	{
 		return false;
 	}
 
 	@ConfigItem(
-		keyName = "notificationMode",
-		name = "Notification mode",
-		description = "Determines mode of notifications",
-		position = 5
+		keyName = "notificationTray",
+		name = "Enable tray notifications",
+		description = "Enables tray notifications",
+		position = 20
 	)
-	default Notifier.NotificationMode notificationMode()
+	default boolean enableTrayNotifications()
 	{
-		return Notifier.NotificationMode.TRAY;
+		return true;
 	}
 
 	@ConfigItem(
-		keyName = "notificationFocused",
-		name = "Send notifications when focused",
-		description = "Toggles idle notifications for when the client is focused",
-		position = 6
+		keyName = "notificationRequestFocus",
+		name = "Request focus on notification",
+		description = "Toggles window focus request",
+		position = 21
 	)
-	default boolean sendNotificationsWhenFocused()
+	default boolean requestFocusOnNotification()
 	{
-		return false;
+		return true;
 	}
 
 	@ConfigItem(
-		keyName = "notificationRequestFocus",
-		name = "Request focus on notification",
-		description = "Toggles window focus request",
-		position = 7
+		keyName = "notificationSound",
+		name = "Enable sound on notifications",
+		description = "Enables the playing of a beep sound when notifications are displayed",
+		position = 22
 	)
-	default boolean requestFocusOnNotification()
+	default boolean enableNotificationSound()
 	{
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "notificationGameMessage",
+		name = "Enable game message notifications",
+		description = "Puts a notification message in the chatbox",
+		position = 23
+	)
+	default boolean enableGameMessageNotification()
```

Comment:
This should be reverted to previous behaviour, there is no reason to change it. The intention of the new notification types was to not use the together with other options, and it should stay that way. Also, this is completely out of scope of this PR?

Reply:
I changed it here because the behavior introduced in #2180 is broken. It does not work. It is in this PR, rather than a separate one because it would be impossible to write this one without depending on it, so there is no reason. If we want to merge 093b11f before this, I will cherry pick it onto a new branch and PR it separately, but I didn't think that was necessary.

Reply:
Based on your use-case, you want only sound with tray notifications. What is also discutable because it looks like it is problem either in Java or in your OS/configuration. But, with custom sound it at least makes sense as separate option and I would do it like that as well. Otherwise I explained the reasoning for the modes, also I dislike pointless toggles, so that is why it is in dropdown and not each type of notification separate. That is also common use-case, so unless there is good reason to change it, it should not be changed.

Reply:
My preferred configuration is [Custom tray/Custom sound/Chat message]. I see no reason other than "I dislike customization" for not having this separate.

Reply:
Well as I said before, your preferred configuration is not useful. Tray notifications are useful when not looking at screen, chat notifications are useful when looking at screen. They server different purposes and they was added under intention to serve for that purposes.

Reply:
I use the chat messages as a way to gauge how much I'm paying attention, as they are moderately temporal.

Reply:
Hmm. Well, I still disagree but I guess. But, this should be different PR on top of this one still so it can be decided there and not block this one.